README.md

```diff
@@ -2,7 +2,7 @@
 
 # hyperimport
 
-⚡ TypeScript imports on steroids. Import C, Rust, Zig, etc in your TypeScript code and more.
+⚡ TypeScript imports on steroids. Import C, Rust, Zig etc. files in your typescript code and more.
```

Comment:
Then, `TypeScript` would fit it now.

Reply:
Right, I also fixed for some other names